scripts/install.sh

```diff
@@ -149,6 +149,9 @@ firezoneSetup() {
   echo "TELEMETRY_ENABLED=$telemEnabled" >> "$installDir/.env"
   echo "TID=$tid" >> "$installDir/.env"
 
+  LATEST_VERSION=$(curl -sL https://api.github.com/repos/firezone/firezone/releases/latest | grep -w tag_name | cut -d '"' -f 4)
```

Comment:
may want to add the `-f` for erroring out if the request fails and `-S` for printing the error even when `-s` is used, e.g. `-fsSL` combined